client/my-sites/plan-features/scroller.jsx

```diff
@@ -0,0 +1,334 @@
+
+		if ( ! styleWeights ) {
+			return null;
+		}
+
+		return (
+			<>
+				<style>
+					{ `.plan-features__header::before { left: ${ -paneWidth - borderSpacing / 2 }px }` }
+				</style>
+				{ styleWeights.map( ( weight, index ) => {
+					const selector = `${ cellSelector }:nth-child(${ index + 1 })`;
+					const opacity = round( weight * ( 1 - MIN_PLAN_OPACITY ) + MIN_PLAN_OPACITY, 2 );
+					let translateX = inRange( weight, 0, 1 ) ? ( 1 - weight ) * 5 : 0;
+
+					if ( translateX && index < visibleIndex ) {
+						translateX = -translateX;
+					}
+
+					const transform = translateX ? `transform: translateX( ${ translateX }% );` : '';
+
+					return (
+						<style key={ selector }>
+							{ `${ selector } { opacity: ${ opacity }; ${ transform } }` }
+						</style>
+					);
+				} ) }
+			</>
+		);
+	}
+
+	renderIndicator( { showIndicator, visibleCount, visibleIndex } ) {
+		const dotClass = 'plan-features__scroll-indicator-dot';
+		const start = visibleIndex;
+		const end = start + visibleCount;
+
+		if ( ! showIndicator ) {
+			return null;
+		}
+
+		return (
+			<div className="plan-features__scroll-indicator">
+				{ range( 0, this.props.planCount ).map( index => (
+					<span
+						key={ index }
+						className={ classNames( dotClass, { 'is-highlighted': inRange( index, start, end ) } ) }
+					/>
+				) ) }
+			</div>
+		);
+	}
+
+	render() {
+		const { children, withScroll, planCount } = this.props;
+
+		if ( ! withScroll ) {
+			return <>{ children }</>;
+		}
+
+		const vars = this.computeStyleVars();
+		const containerClass = classNames( 'plan-features__scroller-container', {
+			'scroll-snap-disabled': this.state.scrollSnapDisabled,
+		} );
+
+		return (
+			/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+			<div className={ containerClass }>
+				{ this.renderStyle( vars ) }
+				<div
+					className="plan-features__scroll-left"
+					style={ { width: vars.paneWidth } }
+					onClick={ this.scrollLeft }
+				>
+					<Button
+						className="plan-features__scroll-button"
+						disabled={ 0 === vars.visibleIndex }
+						onClick={ this.scrollLeft }
+						tabIndex="0"
+					>
+						<Gridicon icon="arrow-left" size={ 24 } />
+					</Button>
+				</div>
+				<div className="plan-features__scroller-wrapper" ref={ this.setWrapperRef }>
+					<div
+						className="plan-features__scroller"
+						style={ { width: vars.scrollerWidth, padding: vars.scrollerPadding } }
+					>
+						{ children }
+					</div>
+				</div>
+				<div
+					className="plan-features__scroll-right"
+					style={ { width: vars.paneWidth } }
+					onClick={ this.scrollRight }
+				>
+					<Button
+						className="plan-features__scroll-button"
+						disabled={ planCount === vars.visibleIndex + vars.visibleCount }
+						onClick={ this.scrollRight }
+					>
+						<Gridicon icon="arrow-right" size={ 24 } />
+					</Button>
+				</div>
+				{ this.renderIndicator( vars ) }
```

Comment:
Not sure if I'm right here, but since these are visual indicators, perhaps we could stack them over the plans instead of under them.

On many screen sizes you don't know they're there at all. 

The other option is to get rid of them.

@Automattic/onboarding-design